pkg/auth/interface.go

```diff
@@ -49,5 +44,7 @@ type GoogleConfig struct {
 }
 
 type OAuthConfig struct {
+	Issuer       string       `json:"issuer"`
+	Audience     string       `json:"audience"`
```

Comment:
Adding these two fields. Before, we used the values from dex. From now, we will set them in the values.yaml